src/api/app/helpers/webui/comment_locks_helper.rb

```diff
@@ -0,0 +1,14 @@
+module Webui::CommentLocksHelper
+  # rubocop:disable Rails/OutputSafety
+  def comment_lock_alert(commentable)
+    alert = 'You can remove the lock by clicking on the button below.'
+    return alert if commentable.comment_lock
+
+    if commentable.is_a?(Package) && commentable.project.comment_lock
+      "You can remove the lock by visiting #{link_to(commentable.project.name, project_show_path(commentable.project))}.".html_safe
```

Comment:
```suggestion
      message = "You can remove the lock by visiting ".html_safe
      message.concat(link_to(commentable.project.name, project_show_path(commentable.project)))
```
I think you can do it like this and avoid hitting the Rails/OutputSafety cop (seen [here](https://www.rubydoc.info/gems/rubocop-rails/2.21.1/RuboCop/Cop/Rails/OutputSafety))

Reply:
Thank you for looking into this. I've addressed it